NBXplorer.Client/DerivationStrategy/DerivationStrategy.cs

```diff
@@ -11,14 +11,52 @@ namespace NBXplorer.DerivationStrategy
 {
 	public class DerivationStrategyOptions
 	{
-		public ScriptPubKeyType ScriptPubKeyType { get; set; }
+
+		public ScriptPubKeyType ScriptPubKeyType
```

Comment:
Don't do this, this is horrible to calculate that everytimes instead of once and affecting ScritPuBKeyType

Reply:
fixed

Reply:
this is not fixed, it should not be calculated.